.github/CODEOWNERS

```diff
@@ -4,6 +4,8 @@
 **/README.md                             @DataDog/documentation @DataDog/agent-integrations
 **/*metadata.csv                         @DataDog/documentation @DataDog/agent-integrations
 **/manifest.json                         @DataDog/documentation @DataDog/agent-integrations
+**/assets/dashboards                     @Datadog/documentation @DataDog/dashboards-backend
+**/assets/monitors                       @Datadog/documentation @DataDog/monitor-app
```

Comment:
Could @DataDog/agent-integrations also be added so we are also pinged when we modify assets?

Reply:
Note: I gave the Write permission to the `@DataDog/dashboards-backend` on this repo 🙂

Reply:
Ah thanks @FlorentClarret I was wondering why I couldn't find them!